_posts/2020-09-08-interacting with ai and automation.md

```diff
@@ -0,0 +1,90 @@
+<br>
+<br>
+
+Anyone who has ever used a BIM software will know that they are rich in tools that allow a high level of detail and creativity. Automation and AI can come in handy to support the design phase of a building. For example, the automatic creation of interconnected elements can help architects complete the most tedious tasks, whereas alerts and suggestions can help them avoid mistakes. 
+
+In the case of the Zuru Tech software, besides the basic principles of architecture, a number of constraints come from factory requirements, which create a preferential path where the user's actions will be guided. 
+
+But when we create user experiences, we have to take into account also different ways of using the software, cases where the ideal scenario (the <i>happy path</i>) will only be partially met. 
+How do you design for those cases? How do you allow the expert user - the highly skilled professional - to still have the freedom to work as they wish and not feel as if the software wants to “teach” them how to do their job, or even force them in a predetermined direction? In short, how do you make sure the software has a high level of adaptability to the user’s needs?
+
+For us, it means giving users choice. For example, users will be able to modify certain automatically created elements and dismiss suggestions that are not in line with what they have set to achieve, as long as the implications are clear (very important!). 
+<br>
+<br>
+
+## 3. Use visual cues
+
+<br>
+<br>
+<a href="/images/miroodles3.png"><img class="blog-image" style="width: 70%; margin:auto; display:block" src="/images/miroodles3.png"></a> 
+<br>
+<br>
+
+Great help in finding the right balance - between showing suggestions that are not overwhelming and making sure the directions are clear - comes from the world of video game design with the use of visual cues. They can take the form of a highlight in a different color to signal an error, a change in the size of an element that can be modified, or a display on hover of an additional function. 
+
+In our case, these strategies work as signifiers for automation algorithms and AI suggestions guided by the rules of architectural design. They help the users discover and learn how to use the various tools of the BIM software interface in a way that is fun and engaging.
+<br>
+<br>
+
+## Conclusions
+
+In conclusion, even though AI and automation are made to simplify our lives, it does not mean interacting with these forms of technology is always straightforward. They have to be carefully studied and adapted with the right amount of signifiers. These signifiers sometimes can come from different fields from the one you are working, so keeping an open mind is crucial for UX designers to find the right solution. 
+
+Thank you for reading so far. If you have any thoughts on this, you can find me on <a href="https://www.linkedin.com/in/martinaloi/">LinkedIn</a> and <a href="https://medium.com/@martina.loi">Medium</a>.
```

Comment:
No need to add references here - they are already visible in the author box automatically added below the article